index.html

```diff
@@ -196,7 +211,7 @@ <h1>About me</h1>
     </div>
   </section>
 
-  <footer class="footer">
+  <footer id="footer">
     <p>I'm always interested in hearing about new projects, 
```

Comment:
- [ ] On the bottom of the footer there are social links, I can see that you made it as a part of the form. This is a bad practice, and it leads to bugs like the icons are not centered in the middle and the divider line is not reaching the right side of the document. I recommend you to add a new div for the bottom footer with icons, and align the content with flex properties making sure that the icons are centered (horizontally and verticaly), the divider stretches from the left side to the right.
Your footer:
<img width="737" alt="Screenshot 2022-04-01 at 19 13 49" src="https://user-images.githubusercontent.com/77038610/161292007-65b27859-b405-4424-8b04-ad8db2576e1c.png">

- [ ] Please also make sure to decrease the height of the icons container it is slightly higher than in Figma.